extensions/blocks/eventbrite/index.js

```diff
@@ -35,6 +35,12 @@ export const icon = {
 	foreground: '#555d66',
 };
 
+export const defaultModalButtonText = _x(
```

Comment:
Is this a reasonable place in this block to define a constant that will be used in multiple places? The pattern seems to be defined here, so I was following it.